packages/accordion/components/Accordion.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { Stack } from "../../styleUtils/layout";
+import { Provider as AccordionProvider } from "./AccordionContext";
+
+interface AccordionProps {
+  /**
+   * Whether multiple accordion items can be open at once
+   */
+  allowMultipleExpanded?: boolean;
+  /**
+   * human-readable selector used for writing tests
+   */
+  ["data-cy"]?: string;
```

Comment:
👏  This is much needed and great to have. Thanks for explicitly supporting `data-cy` throughout the entire component tree.